test/test_filesystem_helper.cpp

```diff
@@ -14,16 +14,42 @@
 
 #include <gtest/gtest.h>
 
+#include <fstream>
 #include <string>
 
 #include "rcpputils/filesystem_helper.hpp"
+#include "temporary_directory_fixture.hpp"
 
 #ifdef _WIN32
 static constexpr const bool is_win32 = true;
 #else
 static constexpr const bool is_win32 = false;
 #endif
 
+namespace
+{
+/**
+ * Create a file with a specific size specified in MiB.
+ *
+ * @param uri Path to the file to create.
+ * @param size File size in MiB
+ */
+void create_file(const std::string & uri, int size)
+{
+  std::ofstream out{uri};
+  if (!out) {
+    throw std::runtime_error("Unable to write file.");
+  }
```

Comment:
use http://www.cplusplus.com/reference/ios/ios/exceptions/ in place of checking `ofstream` after each operation.

Reply:
I should've remembered this!